Core/include/Acts/Vertexing/KalmanVertexTrackUpdater.hpp

```diff
@@ -34,15 +35,16 @@ namespace detail {
 /// refitted track parameters
 ///
 /// @param sMat Track ovariance in momentum space
-/// @param newTrkCov New track covariance matrixs
+/// @param crossCovVP Cross variance between the vertex and
+/// the fitted track momentum
 /// @param vtxWeight Vertex weight matrix
 /// @param vtxCov Vertex covariance matrix
 /// @param newTrkParams New track parameter
-inline BoundMatrix createFullTrackCovariance(const SquareMatrix3& sMat,
```

Comment:
This actually needs to be `inline` since it's defined in the `.ipp`. Even better: move the definition into a `.cpp` file and remove the `inline`.

Reply:
I will just add the inline again as this function will be removed in #2402